app/assets/src/components/Samples.jsx

```diff
@@ -988,24 +989,36 @@ class Samples extends React.Component {
       </div>
     );
 
+    let compare_button_inner = (
```

Comment:
it looks like variable `compare_button_inner` is not being reassigned, so `const` would convey the meaning better :D? Unless I'm missing something, then nvm lol